util/app_server.py

```diff
@@ -0,0 +1,125 @@
+"""Implement logic common to more than one of the Simplified applications."""
```

Comment:
All the stuff in util/ is copied from core and only modified slightly, if at all.